ghost/core/core/server/data/migrations/versions/5.10/2022-08-16-14-25-add-member-created-events-table.js

```diff
@@ -0,0 +1,11 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('member_created_events', {
```

Comment:
Is there a reason we've gone for the singular version of members?

Reply:
Because the word member here is part of the event name, rather than the relation - I can rename it though.

Should we also change it to subscriptions_created_events, or leave that singular?